MNE/disp3D/3DObjects/brain/brainsurfacetreeitem.cpp

```diff
@@ -102,6 +102,10 @@ BrainSurfaceTreeItem::BrainSurfaceTreeItem(int iType, const QString& text)
 
 BrainSurfaceTreeItem::~BrainSurfaceTreeItem()
 {
+    setVisible(false);
+
+    delete m_pRenderable3DEntity;
+    delete m_pRenderable3DEntityActivationOverlay;
```

Comment:
if those members are derived of QObject - consider guarding them with QPointer instead of raw pointer -> delete would not be needed anymore